proto/beacon/p2p/v1/types.proto

```diff
@@ -156,27 +156,22 @@ message ShardAndCommittee {
 
 message BeaconBlock {
   uint64 slot = 1;
-  bytes randao_reveal_hash32 = 2;
-  bytes candidate_pow_receipt_root_hash32 = 3;
   // Ancestor hashes represent a skip list of ancestor beacon block hashes.
   // i'th item is the most recent ancestor whose slot is a multiple of 2^i 
   // for i in the range of 0 to 31.
-  repeated bytes ancestor_hash32s = 4; 
-  bytes state_root_hash32 = 5;
-  // TODO(781): attestations should be a list of attestionRecords according to spec
-  // at ethereum/eth2.0-specs@11d4473. Update this field to AttestionRecord
-  // and remove the AggregatedAttestion message.
-  repeated AggregatedAttestation attestations = 6 [deprecated=true];
-  // repeated AttestationRecord attestations = 6;
-  // Specials consist of exist, penalities, etc.
-  repeated SpecialRecord specials = 7  [deprecated=true]; // Deprecated in favor of unify specials object w/ attestations.
-  repeated bytes signature = 8; // Type of [uint384]?
+  repeated bytes ancestor_hash32s = 2;
+  bytes state_root_hash32 = 3;
+  bytes randao_reveal_hash32 = 4;
+  bytes candidate_pow_receipt_root_hash32 = 5;
+  repeated bytes signature = 6; // Type of [uint384]?
+
   // Block Body
-  BeaconBlockBody body = 9;
+  BeaconBlockBody body = 1000;
```

Comment:
Why 1000? Do we expect a lot of fields in this message?

Reply:
Probably not.. I'll change it back to 9